src/index.ts

```diff
@@ -23,17 +26,16 @@ swdcTracker.initialize = async (swdcApiHost: string, namespace: string, appId: s
 
     swdcTracker.spTracker = tracker([e], namespace, appId, false)
 
-    if (process.env.ENABLE_SWDC_TRACKER === "true") {
-      swdcTracker.testMode = false
-      console.log(`swdc-tracker initialized and ready to send events to ${tracker_api_host}`)
+    if (isTestMode()) {
+      console.log('swdc-tracker test mode on. set env ENABLE_SWDC_TRACKER to "true" to send events');
     } else {
-      swdcTracker.testMode = true
-      console.log('swdc-tracker test mode on. set env ENABLE_SWDC_TRACKER to "true" to send events')
+      console.log(`swdc-tracker initialized and ready to send events to ${tracker_api_host}`);
     }
-    return { status: "success" };
+
+    return success();
   } catch (e) {
     console.log("swdcTracker failed to initialize", e);
```

Comment:
do we still need this console log? can probably remove since were throwing an error

Reply:
no we don't need it, you're right